client/src/modules/inventory/list/modals/log.modal.html

```diff
@@ -38,6 +38,10 @@
   </div>
 </div>
 <div class="modal-footer">
+    <a target='_blank' class="btn btn-primary" data-method="close" href="/inventory/download/log/{{$ctrl.inventory.uuid}}?lang={{$ctrl.lang}}">
```

Comment:
The `data-method=close` will confuse our end to end tests.  Could you remove it?